wire-ios-notification-engine/Sources/NotificationSession.swift

```diff
@@ -362,6 +362,7 @@ extension NotificationSession: PushNotificationStrategyDelegate {
                 WireLogger.notifications.info("ignoring event")
             }
         }
+        context.saveOrRollback()
```

Comment:
What was this solving?

Reply:
Save changes and increase the badge count. If you check the develop branch, the badge count will always be 1.